src/node/handler/PadMessageHandler.js

```diff
@@ -1061,6 +1062,7 @@ const handleClientReady = async (socket, message, authorID) => {
         rev: pad.getHeadRevisionNumber(),
         time: currentTime,
       },
+      accessStatus,
```

Comment:
Can you explain why this is here? CLIENT_READY should iirc not be sent when permissions are not sufficient to edit/join the pad?

Reply:
In the first round of tests I needed it, it is not necessary, I removed it.